src/main/java/io/github/moulberry/notenoughupdates/profileviewer/data/APIDataJson.java

```diff
@@ -55,6 +55,7 @@ public static class Currencies {
 	public static class Profile {
 		public int personal_bank_upgrade = 0;
 		public boolean cookie_buff_active = false;
+		public int bank_account = 0;
```

Comment:
bank_account is sent as a floating point number like purse, it should probably be read as a float aswell.

Reply:
good point, i though i did that, fixing rq